VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -101,65 +101,62 @@
 
             <LinearLayout
                 android:id="@+id/last_access_bar"
-                android:orientation="vertical"
                 style="@style/last_access_bar_style"
-                android:visibility="gone">
```

Comment:
I couldn't find anything in code that guarantees this layout is initially hidden.